builder/amazon/common/step_create_tags.go

```diff
@@ -31,6 +39,22 @@ func (s *StepCreateTags) Run(state multistep.StateBag) multistep.StepAction {
 	for region, ami := range amis {
 		ui.Say(fmt.Sprintf("Adding tags to AMI (%s)...", ami))
 
+		// Convert tags to ec2.Tag format
+		amiTags, err := ConvertToEC2Tags(s.Tags, *ec2conn.Config.Region, sourceAMI, s.Ctx, ui)
```

Comment:
is this getting called twice?

here
* https://github.com/rickard-von-essen/packer/blob/ce411a0d17ae40960ce2954b0fc0fe8680699f45/builder/amazon/common/step_create_tags.go#L42-L56
* https://github.com/rickard-von-essen/packer/blob/ce411a0d17ae40960ce2954b0fc0fe8680699f45/builder/amazon/common/step_create_tags.go#L104-L119

probably a rebase error